client/hijack.go

```diff
@@ -50,6 +50,14 @@ func fallbackDial(proto, addr string, tlsConfig *tls.Config) (net.Conn, error) {
 	return net.Dial(proto, addr)
 }
 
+// HijackDialer returns a dialer to a hijacked connection.
+func (cli *Client) HijackDialer(req *http.Request, proto string, meta map[string][]string) func(context.Context) (net.Conn, error) {
```

Comment:
Either remove `DialSession` now and replace with this or add another method `DialGRPC` like `DialSession`, or add URL to `DialSession`

Reply:
I initially had DialGRPC I can add it back.